notebooks/official/pipelines/google_cloud_pipeline_components_bqml_text.ipynb

```diff
@@ -44,6 +44,7 @@
         "  </td>\n",
```

Comment:
<p>pipeline run is left hanging. Add something to show pipeline produced something.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/822/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>